zerver/models/messages.py

```diff
@@ -105,6 +120,9 @@ class Meta:
 
     @override
     def __str__(self) -> str:
+        if not self.is_channel_message:
+            return f"{self.recipient.label()} /  / {self.sender!r}"
```

Comment:
Have kept this just like this was before, but maybe we can update this to `f"{self.recipient.label()} / {self.sender!r}"` or do we want some special string for topic.

Reply:
Yeah maybe start a #backend thread about it with output from printing these? Probably we want to redo both more fully.